Makefile

```diff
@@ -5,12 +5,12 @@ GOGCFLAGS ?= -gcflags=all="-N -l"
 endif
 
 # ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
-ENVTEST_K8S_VERSION = 1.30.3
+ENVTEST_K8S_VERSION = 1.31
```

Comment:
Does this work without the patch version?

Reply:
It did here: https://github.com/openshift/machine-api-provider-azure/commit/43fec66c8bf4542ee7f03ad54d4411157ea832dd